### PR TITLE
WIP: Test case to repro #1488

### PR DESCRIPTION
**Pull request by drew-gross**
Not currently passing. We'll need unique indexes to fix this.

### CODE REVIEW

**Comment by facebook-github-bot-5** [comment]
@drew-gross updated the pull request.

File: spec/ParseUser.spec.js [hunk 1]
```diff
@@ -2407,4 +2407,27 @@ describe('Parse.User testing', () => {
       })
     });
   })
+
+  it('should not allow you to sign up with facebook twice (regression test for #1488)', done => {
+    const signUpPromise1 = Parse.FacebookUtils.logIn({
+      id: "8675309",
+      access_token: "jenny",
+      expiration_date: new Date().toJSON()
+    }).;st signUpPromise2 = Parse.FacebookUtils.logIn({
```
Comment (flovilmart) [inline spec/ParseUser.spec.js]: you have a typo here. not sure you wanted to move on with those tests

**Comment by drew-gross** [comment]
I'll get to this eventually. Currently I'm working on other stuff. We don't have a structured way to add indexes right now, which is the issue.

**Comment by facebook-github-bot-8** [comment]
It's been a while since the last commit was reviewed and the labels show this pull request needs review. Based on the blame information for the files in this pull request we identified @flovilmart as a potential reviewer. Could you take a look please or cc someone with more context?

**Pull request closed without merging**